src/frontend/src/views/Upload/components/Metadata/components/Table/components/Row/components/LocationField/index.tsx

```diff
@@ -1,75 +1,126 @@
-import { MenuItem } from "czifui";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import { DefaultMenuSelectOption, Dropdown } from "czifui";
 import { FormikContextType } from "formik";
-import React from "react";
+import { escapeRegExp } from "lodash/fp";
+import React, { useEffect, useReducer, useState } from "react";
 import { Metadata } from "src/views/Upload/components/common/types";
 import ApplyToAllColumn from "../common/ApplyToAllColumn";
-import { COUNTIES } from "./COUNTIES";
-import { MenuSubtext, StyledTextField } from "./style";
-
-const SELECT_PROPS = {
-  displayEmpty: true,
-  renderValue,
-};
+import { StyledDiv } from "./style";
 
 interface Props {
   fieldKey: keyof Metadata;
+  accessoryKey: keyof Metadata;
```

Comment:
what does this prop represent, conceptually?

Reply:
That...I don't think I need any more. I'll fix it.